packages/client/src/v2-events/utils.test.ts

```diff
@@ -278,9 +283,10 @@ const testCases = [
       addressType: AddressType.DOMESTIC
     },
     systemVariables: {
-      $user: {
+      user: {
```

Comment:
Renamed to `user` without $ sign because we want to prevent `$user.name` which can be picked like [this](https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/v2-events/utils.ts#L138). Imo, naming it differently could be not ideal (e.g. - `$userField` ) because then that could be used as a new template variable from country-config like `firstname: '$userField.name'` which we don't want in the first place